TFT/src/User/API/LevelingControl.h

```diff
@@ -12,20 +12,31 @@ extern "C" {
 
 typedef enum
 {
-  LEVEL_BOTTOM_LEFT = 0,  // bottom left bed corner
-  LEVEL_BOTTOM_RIGHT,     // bottom right bed corner
-  LEVEL_TOP_RIGHT,        // top right bed corner
-  LEVEL_TOP_LEFT,         // top left bed corner
-  LEVEL_CENTER,           // center bed point
-  LEVEL_POINT_COUNT
+  LEVEL_NO_POINT = -1,
+  LEVEL_BOTTOM_LEFT,    // bottom left bed corner
+  LEVEL_BOTTOM_RIGHT,   // bottom right bed corner
+  LEVEL_TOP_RIGHT,      // top right bed corner
+  LEVEL_TOP_LEFT,       // top left bed corner
+  LEVEL_CENTER,         // center bed point
+  LEVELING_POINT_COUNT
 } LEVELING_POINT;
 
-typedef int16_t LEVELING_POINT_COORDS[LEVEL_POINT_COUNT][2];  // [][0] X coord, [][1] Y coord
-
-void levelingGetPointCoords(LEVELING_POINT_COORDS coords);
-LEVELING_POINT levelingGetPoint(int16_t x, int16_t y);  // return the point matching XY coords or LEVEL_CENTER in case of no match
-void levelingProbePoint(LEVELING_POINT point);
-void levelingMoveToPoint(LEVELING_POINT point);
+typedef int16_t LEVELING_POINT_COORDS[LEVELING_POINT_COUNT][2];  // [][0] X coord, [][1] Y coord
+
+extern LEVELING_POINT probedPoint;  // last probed point or LEVEL_NO_POINT in case of no new updates
+extern float probedZ;               // last Z offset measured by probe
+
+void levelingGetPointCoords(LEVELING_POINT_COORDS coords);   // get all point coords
+LEVELING_POINT levelingGetPoint(int16_t x, int16_t y);       // get point matching XY coords or LEVEL_CENTER in case of no match
+void levelingMoveToPoint(LEVELING_POINT point);              // move to point
+void levelingProbePoint(LEVELING_POINT point);               // probe point
+void levelingSetProbedPoint(int16_t x, int16_t y, float z);  // set probed point and Z offset for point matching XY coords
+inline LEVELING_POINT levelingGetProbedPoint(void)
```

Comment:
"inline" in ".h" files? Why?

Reply:
it is simply a way to access a variable meant to be hidden (if it could be declared in .c) through the API function instead of accessing it directly. It was done in the past also in parseAck with isHostDialog. The data needs to be defined as "extern" in .h but the intent is to use the API function to access it instead of accessing it directly.
You could not declare and use externally an inline function if it is defined in .c

Reply:
> You cannot define and use externally an inline function if it is declared in .c (the scope is inside .c).

I think you can, just try it.

Reply:
"extern" is the opposite of "static" and you cannot use a static inline function outside the .c.

Reply:
My bad, I ignored that they are declared "static".

Reply:
no problem